getting-started/laravel.html.md.erb

```diff
@@ -10,286 +10,160 @@ nav: firecracker
-
-##### clear_env
-
-What does _this_ do? This lets PHP-FPM access the environment variables provided by the system. If you leave this as its default _yes_ you will find Laravel is not happy. It can't access all the environment variables you've either set within the `fly.toml` file (we'll come on to that in a moment) or by using `fly secrets`. And those environment variables are _very_ important, controlling things such as the application environment and secret key.
-
-#### Add a fly.toml
 
-If you have deployed any application to Fly before, you will recognise this. The `fly.toml` file tells Fly about your application, such as what ports it should use, the protocol and health-checks. One is generated for you by the Fly CLI however you may prefer to have one already if you know the settings you need.
+### Deploy
 
-Our `fly.toml` includes some standard settings, such as exposing port `443` and `80` to the outside world, and have our app listening on port `8080`. It has a http healthcheck on `/` which (if all is well) will return a `200` status code. Notice also the `[env]` section which contains environment variables. This contains any variable whose value is not secret.
+Finally, run `fly deploy` to build and deploy your application!
 
-If using our `fly.toml` you will need to update the application name from _fly-hello-laravel_ to your own app name in two places:
-
-```toml
-app = "fly-hello-laravel"
-
-APP_URL = "https://fly-hello-laravel.fly.dev"
-```
-
-**Note:** You should set `APP_DEBUG` as false when running Laravel in production. But we _don't_ set that here within the `fly.toml`. Why? Currently it seems that that value is parsed into a string. This can be confirmed using `gettype(env('APP_DEBUG'))` within Laravel. It returns _string_. And the string _"false"_ is _truthy_. And so debgging remains on. In Laravel's `config/app.php` the default value of `APP_DEBUG` should be false and so it is left as false _unless_ we set it as true.
-
-We also haven't made use of the `[[statics]]` option to [offload serving static assets](https://fly.io/docs/reference/configuration/#the-statics-sections) to Fly. That is because we are using nginx which is already a very efficient way to serve static files. However you could add that if you prefer.
+You should be able to visit `https://your-app-name.fly.dev` and see the Laravel demo home page.
 
-Due to the way Laravel caches files, we don't use the option to cache configuration. We want to still have access to environment variables set at runtime.
+<%= partial "partials/launched" %>
 
-You should also make sure you do not deploy with cached routes. Why? If a different `APP_KEY` was used (as it will likely be) you will get a Laravel error complaining that:
+## _What Just Happened?_
 
```

Comment:
This is good info. I think it should show up before people run `fly launch`.

Reply:
Makes sense! I updated it a bit and added a bit more formatting (bolding things). LMK if you think it's too much info / too confusing up there. Here's what it looks like locally:
<img width="1601" alt="image" src="https://user-images.githubusercontent.com/467411/173135581-58acd871-221d-409c-89ab-d3649c9ee5ca.png">